src/ConnectedMachine/ConnectedMachine.Autorest/test/New-AzConnectedLicenseProfileFeature.Tests.ps1

```diff
@@ -0,0 +1,22 @@
+if(($null -eq $TestName) -or ($TestName -contains 'New-AzConnectedLicenseProfileFeature'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'New-AzConnectedLicenseProfileFeature.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'New-AzConnectedLicenseProfileFeature' {
+    It '__AllParameterSets' -skip {
```

Comment:
Is this test case valid? If so, please remove the -skip.

Reply:
fixed